src/applications/caregivers/actions/fetchFacilities.js

```diff
@@ -3,35 +3,33 @@ import environment from 'platform/utilities/environment';
 import { apiRequest } from 'platform/utilities/api';
 import content from '../locales/en/content.json';
 
-export const fetchFacilities = async (mapBoxResponse, request = null) => {
-  if (mapBoxResponse?.errorMessage) {
-    return mapBoxResponse.errorMessage;
-  }
-
-  const [longitude, latitude] = mapBoxResponse;
-  const radius = 500;
-  const perPage = 500;
-
+export const fetchFacilities = async ({
+  lat,
+  long,
+  radius = 500,
+  page = 1,
+  perPage = 5,
+}) => {
   const lightHouseRequestUrl = `${
     environment.API_URL
-  }/v1/facilities/va?lat=${latitude}&long=${longitude}&radius=${radius}&per_page=${perPage}`;
+  }/v0/health_care_applications/facilities?type=health&lat=${lat}&long=${long}&radius=${radius}&page=${page}&per_page=${perPage}`;
```

Comment:
This endpoint wont be the same once @Mitch-A6 is able to apply the VES filtering, correct? If so, we don't want to be relying on that endpoint. It might suck to have to duplicate the work, but I think it would be a good idea to isolate these applications, since they do not rely on the same services.

Reply:
Noting that this will be revisited at a later date.